src/modules/dashboard/components/ExpenditurePage/Split/SplitEqual/SplitEqual.tsx

```diff
@@ -0,0 +1,221 @@
+import TokenIcon from '~dashboard/HookedTokenIcon';
+import { ValuesType } from '~pages/ExpenditurePage/ExpenditurePage';
+import Numeral from '~core/Numeral';
+import { getTokenDecimalsWithFallback } from '~utils/tokens';
+import UserPickerWithSearch from '~core/UserPickerWithSearch';
+import { filterUserSelection } from '~core/SingleUserPicker';
+import { supRenderAvatar } from '~dashboard/ExpenditurePage/Recipient/Recipient';
+import Icon from '~core/Icon';
+import Button from '~core/Button';
+
+import styles from './SplitEqual.css';
+import { initalRecipient } from '../constants';
+
+const MSG = defineMessages({
+  amountLabel: {
+    id: 'dashboard.ExpenditurePage.Split.SplitEqual.amountLabel',
+    defaultMessage: 'Amount',
+  },
+  recipientsCounterText: {
+    id: 'dashboard.ExpenditurePage.Split.SplitEqual.recipientsCounterText',
+    defaultMessage: 'recipient{s}',
+  },
+  each: {
+    id: 'dashboard.ExpenditurePage.Split.SplitEqual.each',
+    defaultMessage: 'each',
+  },
+  deleteIconTitle: {
+    id: 'dashboard.ExpenditurePage.Split.SplitEqual.deleteIconTitle',
+    defaultMessage: 'Delete recipient',
+  },
+  addRecipientLabel: {
+    id: 'dashboard.ExpenditurePage.Split.SplitEqual.addRecipientLabel',
+    defaultMessage: 'Add recipient',
+  },
+});
+
+const displayName = 'dashboard.ExpenditurePage.Split.SplitEqual';
+
+interface Props {
+  sidebarRef: HTMLElement | null;
+  colony: Colony;
+}
+
+const SplitEqual = ({ colony, sidebarRef }: Props) => {
+  const { formatMessage } = useIntl();
+  const { setFieldValue } = useFormikContext<ValuesType>();
+  const [, { value: recipients }] = useField<
+    { user?: AnyUser; amount?: number; key: string }[]
+  >('split.recipients');
+  const { tokens: colonyTokens } = colony || {};
+  const [, { value: amount }] = useField<{
+    value: string;
+    tokenAddress: string;
+  }>('split.amount');
+  const token = colony.tokens.find(
+    (colonyToken) => colonyToken.address === amount?.tokenAddress,
+  );
+  const { colonyAddress } = colony || {};
+  const { data: colonyMembers } = useMembersSubscription({
+    variables: { colonyAddress: colonyAddress || '' },
+  });
+
+  const selectedToken = useMemo(
+    () =>
+      colonyTokens.find(
+        (colonyToken) => colonyToken.address === amount.tokenAddress,
+      ),
+    [colonyTokens, amount.tokenAddress],
+  );
+
+  const recipientsCount = useMemo(() => {
+    const recip = recipients?.filter(
+      (recipient) => recipient?.user?.id !== undefined,
+    );
+
+    return recip.length || 0;
+  }, [recipients]);
+
+  const calculatedAmount = useMemo(() => {
+    return !amount.value ? 0 : Number(amount?.value) / (recipientsCount || 1);
+  }, [amount, recipientsCount]);
+
+  useEffect(() => {
+    setFieldValue(
+      'split.recipients',
+      recipients.map((recipient) => {
+        return { ...recipient, amount: calculatedAmount };
+      }),
+    );
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why are `recipients` not included in the dependency array? If there is a specific reason, please, add a comment explaining